spec/system/admin/order_cycles/simple_spec.rb

```diff
@@ -714,9 +714,8 @@
     input = find(".flatpickr-calendar.open .flatpickr-minute")
     input.send_keys datetime.strftime("%M").to_s.strip
     input.send_keys :enter
-    within "#save-bar" do
-      expect(page).to have_content "You have unsaved changes"
-    end
+    wait_until { page.find("#save-bar").present? }
```

Comment:
Hm, `within` or `page.find` are waiting for the same amount of time already. In theory, this shouldn't make any difference. If it does then maybe there's a bug in Capybara?

Reply:
I agree. It's hard to say it actually brings improvement - the local 100% pass rate has shown to fail often on the actual build.

Another option could be to increase the `wait` time for this specific example, like:

`page.find("#save-bar", wait: 15) # increases the default wait time from 10 to 15 sec`

What do you think?